src/ooxml/java/org/apache/poi/openxml4j/opc/ZipPackage.java

```diff
@@ -344,7 +344,7 @@ else if (contentType != null) {
             }
         }
 
-        return partList.values().toArray(new ZipPackagePart[partList.size()]);
```

Comment:
They actually are ZipPackageParts at that point, but the method returns PackagePart[] so no consumer can know about that. Seemed odd to force it to cast at that point, though it makes no real difference if you wanna keep it as it was.